pom.xml

```diff
@@ -207,8 +205,35 @@
                     </plugin>
                     <plugin>
                         <groupId>io.fabric8</groupId>
-                        <artifactId>fabric8-maven-plugin</artifactId>
-                        <version>${version.fabric8.maven.plugin}</version>
+                        <artifactId>docker-maven-plugin</artifactId>
+                        <version>${version.docker.maven.plugin}</version>
+                        <configuration>
+                            <images>
+                                <image>
+                                    <name>${docker.name}</name>
+                                    <build>
+                                        <from>fabric8/java-centos-openjdk8-jdk:1.4.0</from>
+                                        <env>
+                                            <AB_ENABLED>jolokia</AB_ENABLED>
```

Comment:
Do we need this necessarily? Or can we turn it off?

Reply:
There is actual an issue with Prometheus agent and Wildfly Swarm, i.e. Wildfly Swarm will not startup. This option switches on Jolokia only. Alternatively you could use `<AB_OFF>true</AB_OFF>` which switches off Jolokia and Prometheus --> https://hub.docker.com/r/fabric8/java-centos-openjdk8-jdk/

I'm using that base image, because I did it ;-) and it includes https://github.com/fabric8io-images/run-java-sh which is a sophisticated Java startup script which respects container contstraints and more. Its really targeted to get Java running smoothely in the cloud.

But we could also use the official java image and use a `java -jar` entrypoint. Not very 'cloudy' but minimalistic. Should I adapt the PR for this ?

Reply:
btw, you can try to remove this env and staart the container afterwards.

This is btw also the same what the wildfly generator in f-m-p does in the background.

Reply:
Do we need `fabric8/java-centos-openjdk8-jdk:1.4.0` or would the JRE be sufficient?

Why do you say using the official java image and `java -jar` is not very cloudy?

Reply:
Can you update to use a plain Java base image?

Also JDK vs JRE?

Reply:
yes, will do. gimme a second.